examples/inference/margtime/run.sh

```diff
@@ -1,6 +1,6 @@
 OMP_NUM_THREADS=1 pycbc_inference \
 --config-file `dirname "$0"`/margtime.ini \
---nprocesses 2 \
+--nprocesses 1 \
```

Comment:
We should have one inference example somewhere in the testsuite run for nprocesses > 1.